HW_7/Task 2.js

```diff
@@ -0,0 +1,27 @@
+// Task 1
+function isPalindrome(str) {
+    return str.split('').reverse().join('') == str;
+}
+console.log(isPalindrome("казак"));
+
+console.log(isPalindrome("кот"));
+
+// Task 2
+const sentence = "I am super engineer ssssssss";
+function getWordsWithMaxLength(sentence) {
+    const words = sentence.split(" ");
+    const result = [];
+    let maxLength = 0;
+    for (const word of words) {
+        if (word.length > maxLength) {
+            maxLength = word.length;
+        }
+    }
+    for (const word of words) {
+        if (word.length === maxLength) {
+            result.push(word);
+        }
+    }
```

Comment:
Можем делать все в одном цикле
```suggestion
    for (const word of words) {
        if (word.length > maxLength) {
            maxLength = word.length;
        }
    }
    for (const word of words) {
        if (word.length === maxLength) {
            result.push(word);
        }
    }
```